src/webapp/README.md

```diff
@@ -0,0 +1,118 @@
+| **Redirect URI**             | `http://localhost:5000/auth/redirect`                                        | Required value for this sample.                                                                    |
+| **Client secret**            | _Value shown in Azure portal_                                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **Implicit grant & hybrid flows** | _None selected_                                                         | This sample does not use the implicit grant or hybrid flows.                                       |
+| **Allow public client flows** | **No**                                                                      | This sample does not use a public client flow.                                                     |
+| **Token configuration**      | _No additional claims_                                                       | This sample does not rely on any additional claims existing in the tokens.                         |
+| **App roles**                | _Add an App role called `admin` for use by **Users/Groups**_                 | Required value for this sample. <bn /> One route requires your user to be assigned this role.      |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+Use these settings in your Enterprise Application for this sample app
+
+| Enterprise Application <br/> setting | Value for this sample app                                            | Notes                                                                                              |
+|:------------------------------------:|:---------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Users and groups**                 | _Add yourself with the **Role** of **admin**_                        | Required value for this sample. <bn /> One route requires your user to be assigned this role.      |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample with app registration values
+
+Open the _default\_settings.py_ file and modify the three Azure Active Directory configuration properties using the values from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app).
+
+- `CLIENT_ID`: `"Application (client) ID"`
+- `AUTHORITY`: `"https://login.microsoftonline.com/{your tenant}"`
+- `CLIENT_CREDENTIAL`: `"********"`
```

Comment:
As noted in other comment, use single-line comments and put these in one Python-tagged code snippet.

Reply:
Where will this python-tagged code snippet be?  I haven't added snippets yet as we agreed that the content developer can be responsible for marking up the code so that it fits their narrative flow -- but I can put this wherever you want in anticipation.  Sorry I'm not following the suggestion for "single-line" comments and the interplay between the README (your other suggestion) and now Python-tagged code (this suggestion)

Reply:
Let me know if the new README + config + code layout works for you now in this regard.